unstructured/ingest/interfaces.py

```diff
@@ -11,12 +12,41 @@
 from unstructured.staging.base import convert_to_dict
 
 
+@dataclass
+class StandardConnectorConfig:
+    """Common set of config options passed to all connectors."""
+
+    # where raw documents are stored for processing, and then removed if not preserve_downloads
+    download_dir: str
+    # where to write structured data outputs
+    output_dir: str
```

Comment:
these being required probably made more sense when they were standard but could be omitted. maybe we make this Optional[str] = None so you don't have to define it. also noting, these are in the weird "standard" place where they are standard to everything BUT local (local is actually the biggest outlier). tempted to carve out as some sort of "output" mixin, but this would literally just be carving around the local connector afaict.

Reply:
yeah, seems fine as is right now, but could definitely revisit.